infra/modules/frontend/browser/static_website.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_s3_bucket" "gatsby_static_bucket" {
+  bucket = "gatsby_static_bucket_${var.deployment_stage}"
+
+  website {
+    index_document = "index.html"
+    error_document = "404.html"
+  }
+}
+
+resource "aws_s3_bucket_public_access_block" "gatsby_static_bucket_publicaccess" {
+  bucket = "${aws_s3_bucket.gatsby_static_bucket.id}"
+
+  block_public_acls = false
+  block_public_policy = false
+  ignore_public_acls = false
+  restrict_public_buckets = false
+}
+
+data "aws_iam_policy_document" "gatsby_static_bucket_policy_document" {
+  statement {
+    actions = [
+      "s3:GetObject"
+    ]
+
+    resources = [
+      "${aws_s3_bucket.gatsby_static_bucket.arn}/*"
+    ]
+
+    principals {
```

Comment:
Why is the principals set to * here?

Reply:
Otherwise the site would not be publicly accessible.